README.md

```diff
@@ -19,413 +19,55 @@ Our main goal is to enable web developers to integrate MathType on the widest ra
-
-### How to update dependencies
-
-**1. Prepare the environment.**
-
-```sh
-  # Install this project dependencies, in case you didn't already.
-  $ npm install
-  # You may need to run the clean command, if you executed the start command previously.
-  $ npm run clean-all
-```
-
-**2. Generate the Javascript files**
-
-Run the 'moodle' command, first. And compile the `mathtype-html-integration-devkit` package.
+## Features
 
-```sh
-  $ npm run clean-all
-  # Run the Moodle generation command:
-  # It will run the lerna bootstrap command and then compile the packages
-  # using the 'moodle' parameter. See 'scripts/services/README.md' for more
-  # details.
-  # npm run clean => lerna bootstrap => lerna run compile -- moodle
-  $ npm run moodle
-  cp output/moodle-tinymce4/plugin.min.js packages/mathtype-tinymce4/plugin.min.moodle.js
+- **Ease of use**. Write equations with an interface that provides a user-friendly experience from day one; forget about having to learn LaTeX to write math on a computer.
+- **Professionality**. MathType is an online equation editor that provides perfect quality math formulas.
+- **Handwriting**. Working on a touch device? No worries, MathType will convert your handwritten formula into a clean digital equation ready to be used in your documents.
+- **Accessibility**. MathType is compliant with accessibility requirements to create software for individuals with disabilities.
 
-  # Since 'lerna bootstrap' has been run, it's the turn to update 'Atto' editor.
-  cd packages/mathtype-html-integration-devkit
-  npm run build
-```
+## Supported editors
 
-**3. Copy the files to the Moodle plugins**
+We develop and maintain MathType Web plugins for [CKEditor], [Froala], and [TinyMCE].
 
-Last step consists on updating the Third Party library dependency files on the Moodle plugin source code.
+We also offer a generic integration for a plain HTML5 editable `div`.
+You can adapt this integration to use MathType in the web editor of your choice.
 
-```sh
-  # 'TinyMCE' editor.
-  # {MOODLE_TINYMCE_PLUGIN} is the path for the Moodle plugin source code.
-  cp packages/mathtype-tinymce4/plugin.min.moodle.js {MOODLE_TINYMCE_PLUGIN}/tinymce/editor_plugin.js
-  # 'Atto' editor.
-  # {MOODLE_ATTO_PLUGIN} is the path for the Moodle plugin source code.
-  cp packages/mathtype-html-integration-devkit {MOODLE_ATTO_PLUGIN}/core.js
+For the MathType integration for Moodle and other LMS see [here](https://www.wiris.com/es/solutions/education/).
 
-```
+## Demos
 
-## Examples for developers
+In order to manually try out each plugin, there's a set of developer demos on the [`demos`](demos) folder.
 
-In order to manually test each plugin, there's a set of technical demos on
-the ['demos/'](./demos/) folder on this project.
+Refer to the [documentation](docs/demos/README.md) for more information.
 
-Refer to the [README](demos/README.md) file for more information.
+## Documentation
 
-## Technical Support
+- [Introduction](docs/README.md)
```

Comment:
I would add more information about what they can find here, telling we'll show how to clone the repo in different OS, the bootstraing, etc